pubspec.yaml

```diff
@@ -1,14 +1,14 @@
 name: miquido_flutter_lints
 description: Collection of Flutter lints that we use and follow in Miquido
-version: 0.3.4
+version: 0.3.5
 homepage: https://www.miquido.com
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
-  flutter: ^3.3.4
+  sdk: ">=2.18.0 <3.0.0"
+  flutter: ">=3.3.5"
 
 dependencies:
-  dart_code_metrics: ^4.20.0
+  dart_code_metrics: ^4.21.2
```

Comment:
How about removing `^` ?

Reply:
Interesting: 
https://dart.dev/tools/pub/dependencies#caret-syntax
https://dart.dev/tools/pub/dependencies#traditional-syntax

Reply:
So technically it is better leave it as it is as _Specifying dependencies with version ranges is such as ^1.6.3 is a good practice because it allows the pub tool to select newer versions of the package when they become available._

Reply:
_A concrete version number pins the dependency to only allow that exact version. Avoid using this when you can because it can cause version lock for your users and make it hard for them to use your package along with other packages that also depend on it._